src/Components/Sliders/Popular/SliderPopular.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {NavLink} from 'react-router-dom'
+import './SliderPopular.css'
+import ProductCard from '../../ProductCard/ProductCard';
+
+const SliderPopular = (props) => {
```

Comment:
1 please use destruction of arguments https://codeburst.io/es6-destructuring-the-complete-guide-7f842d08b98f
2, please add propTypes https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
1. Will do later
2. Done